clients/pkg/promtail/targets/file/filetargetmanager_test.go

```diff
@@ -44,7 +44,7 @@ func newTestPositions(logger log.Logger, filePath string) (positions.Positions,
 	// Set the sync period to a really long value, to guarantee the sync timer never runs, this way we know
 	// everything saved was done through channel notifications when target.stop() was called.
 	pos, err := positions.New(logger, positions.Config{
-		SyncPeriod:    10 * time.Second,
+		SyncPeriod:    60 * time.Second,
```

Comment:
Should this period stay?

Reply:
Yes, we don't want the sync timer to be run.